src/fhir.js

```diff
@@ -4,7 +4,15 @@ const fetch = require('node-fetch');
 const debug = require('debug')('vsac'); // To turn on DEBUG: $ export DEBUG=vsac
 const { Code, ValueSet } = require('cql-execution');
 
-async function downloadValueSet(apiKey, oid, version, output, vsDB = {}, caching = true) {
+async function downloadValueSet(
+  apiKey,
+  oid,
+  version,
+  output,
+  vsDB = {},
+  caching = true,
+  options = { parseCodeSystem: 'replace' }
```

Comment:
Since this option only pertains to SVS, it might be misleading to put a default here in the FHIR implementation. But since we are trying to have the signatures mirror each other (between fhir.js and svs.js), it probably does make sense to put `options` in the signature -- so we don't ever mistakenly put something else there!

Do you think that something like this might be better?
```js
options = { /* reserved for future use */ }
```
or:
```js
options = { /* only used in SVS */ }
```

Reply:
Yeah that is fair and makes sense.